[2023-01-17T20:42:54.101+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_data_etl_pipeline.extract_integrate_dataset manual__2023-01-17T20:42:50.452360+00:00 [queued]>
[2023-01-17T20:42:54.122+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: accidents_data_etl_pipeline.extract_integrate_dataset manual__2023-01-17T20:42:50.452360+00:00 [queued]>
[2023-01-17T20:42:54.123+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-01-17T20:42:54.124+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2023-01-17T20:42:54.125+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-01-17T20:42:54.182+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): extract_integrate_dataset> on 2023-01-17 20:42:50.452360+00:00
[2023-01-17T20:42:54.197+0000] {standard_task_runner.py:55} INFO - Started process 3053 to run task
[2023-01-17T20:42:54.205+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'accidents_data_etl_pipeline', 'extract_integrate_dataset', 'manual__2023-01-17T20:42:50.452360+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp0x5xby9b']
[2023-01-17T20:42:54.209+0000] {standard_task_runner.py:83} INFO - Job 20: Subtask extract_integrate_dataset
[2023-01-17T20:42:54.337+0000] {task_command.py:389} INFO - Running <TaskInstance: accidents_data_etl_pipeline.extract_integrate_dataset manual__2023-01-17T20:42:50.452360+00:00 [running]> on host 7fc7752d3cb4
[2023-01-17T20:42:54.848+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=accidents_data_etl_pipeline
AIRFLOW_CTX_TASK_ID=extract_integrate_dataset
AIRFLOW_CTX_EXECUTION_DATE=2023-01-17T20:42:50.452360+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-17T20:42:50.452360+00:00
[2023-01-17T20:42:54.868+0000] {logging_mixin.py:137} INFO - got file paths successfully
[2023-01-17T20:42:59.316+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/***/operators/python.py:192 DtypeWarning: Columns (10) have mixed types.Specify dtype option on import or set low_memory=False.
[2023-01-17T20:43:09.714+0000] {logging_mixin.py:137} INFO - loaded CSVs files to df successfully
[2023-01-17T20:43:29.054+0000] {logging_mixin.py:137} INFO - data integrated successfully
[2023-01-17T20:43:29.192+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-17T20:43:29.705+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=accidents_data_etl_pipeline, task_id=extract_integrate_dataset, execution_date=20230117T204250, start_date=20230117T204254, end_date=20230117T204329
[2023-01-17T20:43:30.256+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-01-17T20:43:30.788+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
